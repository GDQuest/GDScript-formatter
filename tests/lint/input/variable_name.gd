#region Good

var good_variable = "good"
var _private_variable = "also good"

var PreloadName = preload("res://SomeResource.tres")
var LoadName = load("res://SomeOtherResource.tres")

#endregion

#region Bad

var badVariableName = "bad"
var BAD_VARIABLE_NAME = "also bad"
var Bad_Variable_Name = "very bad"

#endregion
