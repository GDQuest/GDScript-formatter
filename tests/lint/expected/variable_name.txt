tests/lint/input/variable_name.gd:13:variable-name:error: Variable name 'badVariableName' should be in snake_case or _private_snake_case format
tests/lint/input/variable_name.gd:14:variable-name:error: Variable name 'BAD_VARIABLE_NAME' should be in snake_case or _private_snake_case format
tests/lint/input/variable_name.gd:15:variable-name:error: Variable name 'Bad_Variable_Name' should be in snake_case or _private_snake_case format